geospatial_map/v1/geospatial_map.yaml

```diff
@@ -0,0 +1,424 @@
+          example: Ready
+        message:
+          description: Human-readable explanation of the result for debugging purpose only. This field is required when the result value is `Error`.
+          type: string
+          example: |-
+            Unable to download the dataset https://caa.example.com/geozones.json. Connection refused.
+
+    Position:
+      type: object
+      required:
+        - uomDimensions
+        - verticalReferenceType
+      properties:
+        uomDimensions:
+          $ref: './ed269.yaml#/components/schemas/UomDimensions'
+        verticalReferenceType:
+          $ref: './ed269.yaml#/components/schemas/VerticalReferenceType'
+        height:
+          description: >-
+            Height above vertical reference datum indicated in `verticalReferenceType`, in units of `uomDimensions`.
+          type: number
+          default: 0
+        longitude:
+          description: >-
+            Longitude, degrees east of prime meridian.
+          type: number
+          default: 0
+        latitude:
+          description: >-
+            Latitude, degrees north of the equator.
+          type: number
+          default: 0
+
+    GeospatialMapQueryRequest:
+      type: object
+      required:
+        - checks
+      properties:
+        checks:
+          type: array
+          items:
+            $ref: '#/components/schemas/GeospatialMapCheck'
+
+    GeospatialMapCheck:
+      type: object
+      required:
+        - filterSets
+      properties:
+        filterSets:
+          description: >-
+            Select geospatial features which match any of the specified filter sets.
+          type: array
+          items:
+            $ref: '#/components/schemas/GeospatialFeatureFilterSet'
+
+    GeospatialFeatureFilterSet:
+      description: >-
+        Set of filters to select only a subset of geospatial features.  Only geospatial features which are applicable to all specified filters within this filter set should be selected.
+      type: object
+      properties:
+        position:
+          description: >-
+            If specified, only select geospatial features encompassing this position.
+          anyOf:
+            - $ref: '#/components/schemas/Position'
+        volumes4d:
+          description: >-
+            If specified, only select geospatial features at least partially intersecting one or more of these volumes.
+          type:
+            array
+          items:
+            $ref: './astm_f3548_21.yaml#/components/schemas/Volume4D'
+        after:
+          description: >-
+            If specified, only select geospatial features which encompass at least some times at or after this time.
+          type: string
+          format: date-time
+        before:
+          description: >-
+            If specified, only select geospatial features which encompass at least some times at or before this time.
+          type: string
+          format: date-time
+        restriction_source:
```

Comment:
The basic problem prompting the addition of this field is that apps may use data sources outside the ones we're trying to test.  For instance, a hypothetical BlimpFlyer app may include both the CAA-provided safety data and some restrictions around pointy trees or objects where flying a blimp specifically is unwise.  So, BlimpFlyer will restrict flights more than an app which only includes the CAA-provided safety data.  But, we don't want BlimpFlyer to fail tests, even in the face of requirements like FAA's LAANC where apps are required to indicate whether a LAANC approval specifically is available.  This field is the attempted solution to that problem: when querying the BlimpFlyer USS, we'll ask for just restrictions from the CAA, which should exclude any of its pointy-object restrictions.

So, I would generally expect there to be just one restriction source associated with a set of rules (e.g., FAA LAANC) that we're trying to test, as opposed to one per data source.  I would expect the separation of restrictions into data sources shouldn't have any impact on the performance-based outcomes of comprehending the restrictions in those data sources.  But, I definitely agree it could be used that way if testing at that level of granularity were useful to verify requirements.